### PR TITLE
Addressing command line error from issue #49

### PR DESCRIPTION
**Pull request by nam20485**
Addressing issue #49:

1. add bash .sh script setup-linux.sh that executes the relevant commands originally described in /transDjango/README.md (https://github.com/hackoregon/transportation-backend/blob/master/transDjango/README.md)

2. fix the command line erro regarding psycopg2 python module compilation resulting from executing the command line steps described in the README.md

*(command line steps call for installing python module 'psycopg2' but that command attempts to build the module thus requiring the appropriate c++ libraries are present on the linux system or the compilation step will fail. These include python3-dev and libpq-dev and must be installed via apt prior to running the pip install step.)*

### CODE REVIEW

**Comment by znmeb** [comment]
1. How do we test this? How do we deploy it once it's tested?
2. There's a comment in the script about Linux Mint 18.2. Where are we using Linux Mint 18.2? The deployed system runs on Ubuntu 16.04.x LTS, doesn't it?

**Comment by nam20485** [comment]
I don't believe that there is anything to deploy for this PR, as the script I provided is not intended to be a part of the deployment package on the target system. The .sh script file was only added as a way to capture the changes needed to the command line steps @MikeTheCanuck included in his issue report. The process he described in the issue involves him manually entering and running the steps on the command line from the /transDjango/README.md (they are listed in the issue description), resulting in an error from one of the steps. I would expect when he repeats this process to attempt to verify resolution of the issue he could either read from the script and manually type the commands in or optionally run the script and see if that resolves the error he was encountering and allowing him to complete the steps in the README.md.

In other words, this script does not modify the deployment process or payload, it addresses an error in the README.md. So testing it would involve verifying the steps from the script resolve the error encountered when attempting to perform those steps and if resolved, updating the README.md in the repo.

**Comment by nam20485** [comment]
I should probably have introduced myself by now. 😄 You may or may not remember me from the first half of the transportation 2016 project team (I made a commit or two).  Also I help Bill out by TA'ing the current HackU devops course.

**Comment by znmeb** [comment]
Sounds good - I'll merge it and add you to the team.
